### PR TITLE
feat(ui5-segmented-button): accessibleName property implemented

### PR DESCRIPTION
**Pull request by unazko**
Fixes: #4000

### CODE REVIEW

**Review by elenastoyanovaa — DISMISSED** [review]
Could you please also include someone from your team :) ?

File: packages/main/test/samples/SegmentedButton.sample.html [hunk 1]
```diff
@@ -12,14 +12,14 @@ <h2 class="control-header">SegmentedButton</h2>
 <section>
 	<h3>Basic SegmentedButton</h3>
 	<div class="snippet">
-        <ui5-segmented-button>
+        <ui5-segmented-button accessible-name="Basic SegmentedButton">
```
Comment (ilhan007) [inline packages/main/test/samples/SegmentedButton.sample.html:15]: Not sure if we should add this to every sample, it bloats the markup, you can have a additional section to emphasise on this property if needed, but setting it everywhere is too much

**Review by ilhan007 — DISMISSED** [review]
I left one comment about the sample page, also as Eli said please involve a team mate for review.

**Review by tsanislavgatev — DISMISSED** [review]

File: packages/main/test/samples/SegmentedButton.sample.html (hunk 1 above)
Comment (unazko) [reply packages/main/test/samples/SegmentedButton.sample.html]: Done

File: packages/main/src/SegmentedButton.js [hunk 2]
```diff
@@ -25,7 +25,20 @@ const metadata = {
...
 	languageAware: true,
-	properties: /** @lends sap.ui.webcomponents.main.SegmentedButton.prototype */  {},
+	properties: /** @lends sap.ui.webcomponents.main.SegmentedButton.prototype */  {
+		/**
+		 * Sets the accessible aria name of the component.
+		 *
+		 * @type {String}
+		 * @defaultvalue: ""
+		 * @public
+		 * @since 1.0.1
```
Comment (fifoosid) [inline packages/main/src/SegmentedButton.js:35]: 1.0.3

**Review by tsanislavgatev — APPROVED** [review]

**Review by fifoosid — APPROVED** [review]

**Pull request merged**